app/forms/waste_exemptions_engine/renewal_start_form.rb

```diff
@@ -2,8 +2,21 @@
 
 module WasteExemptionsEngine
   class RenewalStartForm < BaseForm
+    attr_accessor :temp_renew_without_changes
+
+    def initialize(registration)
+      super
+      self.temp_renew_without_changes = @transient_registration.temp_renew_without_changes
+    end
+
     def submit(params)
-      super({}, params[:token])
+      # Assign the params for validation and pass them to the BaseForm method for updating
+      self.temp_renew_without_changes = params[:temp_renew_without_changes]
+      attributes = { temp_renew_without_changes: temp_renew_without_changes }
+
+      super(attributes, params[:token])
```

Comment:
I see that this is a recurring pattern in our forms. And it does feel quite a tangled situation. I have seen the rest of the forms, and with situations like this: https://github.com/DEFRA/waste-exemptions-engine/blob/master/app/forms/waste_exemptions_engine/address_lookup_form.rb#L19:L29 I can see is quite an obliged choice. Maybe we can have some fun at our next dev session and do a 3-peers pairing and try to find a more elegant solution. Would be fun to share some brain power! :D

Reply:
Agreed, it is really clunky! Would be great to sort it out.